src/Api/Models/Response/ConfigResponseModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bit.Core.Models.Api;
+using Bit.Core.Utilities;
+
+namespace Bit.Api.Models.Response
+{
+    public class ConfigResponseModel : ResponseModel
+    {
+        public ConfigResponseModel(string obj = "config") : base(obj)
+        {
+            this.Version = CoreHelpers.GetVersion();
+            this.Server = new ServerConfigResponseModel();
+            this.Environment = new EnvironmentConfigResponseModel();
```

Comment:
I don't believe we typically prefix with `this`.

And should we just omit the `Server` in this case?
Do we need to provide the environment values?

Reply:
Good points. I'll remove the `this`keywords, and will omit initializing `Server`.

I give `Environment` default values here so it's easy to set in the `ConfigController` class, but we can also remove this here and initialize it within the `ConfigController` class if needed. The main reason I am returning the object with a value in the response is for QA purposes, as discussed previously.

Reply:
Based on other comments and updated environment settings that we now return, resolving this.